platform/security/rest/security-rest-clientapi/src/main/java/org/codice/ddf/cxf/client/SecureCxfClientFactory.java

```diff
@@ -42,6 +42,13 @@
    */
   WebClient getWebClient();
 
+  /**
+   * Returns the WebClient
+   *
+   * @return
+   */
+  WebClient getWebSystemClient();
```

Comment:
❓ Why did you need to add this? It looks like it isn't used anywhere.

Reply:
It will not be used in DDF but downstream distributions will use it.